Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:2
+
+ADD . /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
I prefer to use non-specific, somewhat common dir names instead. This is the dir name I've seen commonplace, for example in the Docker Python official image ```onbuild``` tags. Not sure if it's a best practice somewhere or just coincidence, since I'm not a very experienced "pythonist" :)
Also, I think that it is an obvious first try if you're jumping into a running container and trying luck when looking for the app dir.